pages/index.tsx

```diff
@@ -434,8 +434,8 @@ export default function Home() {
               </p>
               <p>That&apos;s enough to complete all the tasks in the game.</p>
               <p>
-                Don&apos;t see your tokens? Double check that your wallet
-                is connected to the Polygon Network.
+                Don&apos;t see your tokens? Double check that your wallet is
```

Comment:
seems like this change shouldn't have happened

Reply:
It looks like a lint fix.